shaas.go

```diff
@@ -88,6 +90,28 @@ func handleExit(res http.ResponseWriter, req *http.Request) {
 	os.Exit(code)
 }
 
+func handleHealth(res http.ResponseWriter, req *http.Request) {
+	responseCode := 200
+	var err error
+	if c := req.URL.Query().Get("responseCode"); c != "" {
+		if responseCode, err = strconv.Atoi(c); err != nil {
+			responseCode = 200
```

Comment:
what about only assigning when there is no error to avoid repeating `200`?

Reply:
this entire file makes me want to refactor it but... baby steps :D  I'll split out the parsing of this in general, I  was mostly imitating the existing style